tests/common-tests/src/main/java/example/sync/AbstractApp.java

```diff
@@ -95,6 +91,7 @@ void shouldFetchOwnerByName() {
 
     @Test
     @Order(4)
+    @Disabled("Getting Flux (findAll) is failing using hibernate-reactive 2.0")
```

Comment:
These tests are temp disabled for hibernate-reactive

Reply:
This is fixed now, hibernate-reactive folks suggested improved code that works ok now.